superset-frontend/src/SqlLab/components/ResultSet/index.tsx

```diff
@@ -213,6 +219,28 @@ export default class ResultSet extends React.PureComponent<
     }
   }
 
+  async createExploreResultsOnClick() {
+    const { results } = this.props.query;
+    // There is currently redux / state issue where sometimes a query will have serverId
```

Comment:
This comment is probably no longer relevant - did you encounter this issue at all in this workflow?

Reply:
No I didn't, but wasn't sure so kept it in. Will delete.

Reply:
changed